### DISCUSSION TITLE
화면 오른쪽 아래 부분에 Service 더늘릴 수 있나요?

### DISCUSSION THREAD

**Discussion by syh-RYAN** [morethanmin/morethan-log#129] [Q&A]
안녕하세요 :)!!

혹시 다름이 아니라 화면 오른쪽 아래 부분에 Service 부분을 더 늘릴 수 있나요??

하나 더 넣으니깐 안보여서 여쭈어봅니다!

**Comment by morethanmin** [comment]
@syh-RYAN 님, 안녕하세요. 우선 문의 주셔서 감사합니다 ;)

확인해보니 `ServiceCard` 컴포넌트에서 배열의 하나의 값만 가져오고 있네요. 
이 부분 수정해주면 좋을 것 같아요. 아래와 같이 크게 두가지 작업이 필요해 보입니다.

- `site.config.js` 에서 `projects`명칭을 `services`로 변경.
-  `ServiceCard` 컴포넌트에서 `map`을 통해 여러 개 표시될 수 있도록 로직 변경.

가능하시다면 @syh-RYAN 님이 PR 올려주셔도 좋고, 조금 어려우시다면 빠른시일 내에 제가 수정해두겠습니다. ㅎㅎ
